bin/create-ebs-volume

```diff
@@ -252,7 +262,7 @@ function create_and_attach_volume() {
               --region $region \
               --availability-zone $availability_zone \
               $volume_opts \
-              --tag-specification "ResourceType=volume,Tags=[{Key=source-instance,Value=$instance_id},{Key=amazon-ebs-autoscale-creation-time,Value=$timestamp}]" \
+              --tag-specification "ResourceType=volume,Tags=[$instance_tags,{Key=source-instance,Value=$instance_id},{Key=amazon-ebs-autoscale-creation-time,Value=$timestamp}]" \
```

Comment:
Is it also apply to the root volume?

Reply:
@zachigene 
no, root volumes should be tagged already by the instance creation
This will tag only the mounted ebs(s)